src/webots/control/WbLanguageTools.cpp

```diff
@@ -69,12 +69,11 @@ QString WbLanguageTools::pythonCommand(QString &shortVersion, const QString &com
     "To fix the problem, you should set the full path of your python command in "
     "Webots->preferences->python command.\n";
 #else
-    QObject::tr("Webots requires Python version 3.10, 3.9, 3.8 or 3.7 from python.org in your current PATH.\n"
+    QObject::tr("Webots requires Python version 3.7 or ealier in your current PATH.\n"
```

Comment:
```suggestion
    QObject::tr("Webots requires Python version 3.7 or newer in your current PATH.\n"
```

I think 'earlier' here would mean 'previous' instead of 'more recent'.